docs/node-mixin/dashboards.jsonnet

```diff
@@ -1,6 +1,6 @@
 local dashboards = (import 'mixin.libsonnet').grafanaDashboards;
 
 {
-  [name]: dashboards[name]
+  [name]: dashboards[name] + { uid: std.md5(name) },
```

Comment:
The rebase may have erased some of your intended changes 😩

Reply:
fixed